Dockerfile-Dev

```diff
@@ -0,0 +1,41 @@
+# ----- Base Java - Check Dependencies ----
+FROM maven:3.6.1-jdk-8 AS base
+ENV APPDIR=/app
+WORKDIR $APPDIR
+ADD pom.xml $APPDIR
+
+### install azure cli
+RUN apt-get update && \
+    apt-get install -y apt-transport-https ca-certificates curl software-properties-common libssl-dev git wget nano lsb-release jq && \
+    curl -sL https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > /etc/apt/trusted.gpg.d/microsoft.asc.gpg && \
+    CLI_REPO=$(lsb_release -cs) && \
+    echo "deb [arch=amd64] https://packages.microsoft.com/repos/azure-cli/ ${CLI_REPO} main" > /etc/apt/sources.list.d/azure-cli.list && \
+    apt-get update && \
+    apt-get install -y azure-cli
+
+#
+# ----Build App with Dependencies ----
+FROM base AS dependencies
+RUN mvn verify clean --fail-never
+ADD . $APPDIR
+RUN mvn clean package -DskipTests
+
+#
+# ---- Release App ----
+FROM  openjdk:8-jre-alpine AS release
+ENV APPDIR=/app
+WORKDIR $APPDIR
+
+# Create a user
+RUN addgroup -g 4120 -S helium && \
+    adduser -u 4120 -S helium -G helium && \
+    mkdir -p /home/helium && \
+    chown -R helium:helium /home/helium
+USER helium
+
+#Note: Every time we update helium version, we must update the jar version below
```

Comment:
need to remove this manual step. It's not really ci-cd if you have to edit every time ...